### DISCUSSION TITLE
Scrapping Data from BookBeat

### DISCUSSION THREAD

**Discussion by venyxal** [advplyr/audiobookshelf#1464] [General]
Hi,
it would be nice to get data and infos from https://www.bookbeat.de

**Comment by advplyr** [comment]
Is there a public API?

**Comment by Hallo951** [comment]
https://api.bookbeat.com/api/books/41/601608

**Reply by advplyr** [reply]
Docs?

**Reply by Hallo951** [reply]
I don't know. I just stumbled across it earlier. ;)

**Reply by advplyr** [reply]
It doesn't look like this is public so we probably won't be implementing it.
